integration_testing/features/app-context/app-context-disable-device-network-access.feature

```diff
@@ -0,0 +1,16 @@
+Feature: Disabled network access to Kolibri app server
+	If the network access to Kolibri server app is disabled, users who navigate to its URL need to see the notification message
+
+Background:
+	Given that the device setting *Allow others in the network to access Kolibri on this device using a browser* is unchecked
+		And I use an external device to connect to the Kolibri app server and access Kolibri
+
+Scenario: Attempt to connect to the Kolibri app server from an external devices
+  When I type the Kolibri app server URL address on my external device browser
+  Then I see this message on the Kolibri sign in page: *Access to Kolibri has been restricted for external devices. To change this, sign in as a super admin and enable 'Allow others in the network to access Kolibri on this device using a browser', located in Device settings*
+
+Scenario: Security alert when network access setting is enabled
+	When I enable the device setting *Allow others in the network to access Kolibri on this device using a browser*
```

Comment:
All this setting is not visible if you are not in an app context
In the last design change this was always visible, not when the setting is enabled. If that's how it should be, please fill an issue for it to be changed. Currently the security alert is always visible.

Reply:
> All this setting is not visible if you are not in an app context

This scenario is app-specific (goes to the separate folder), so if I understand correctly it applies.

> In the last design change this was always visible, not when the setting is enabled. If that's how it should be, please fill an issue for it to be changed. Currently the security alert is always visible.

Are the changes in #7051 meant to implement this new _visibility_?